docs/can-guides/commitment/recipes/credit-card-simple/credit-card-simple.md

```diff
@@ -31,11 +35,15 @@ To use the widget:
    the invalid inputs should be highlighted red, and the _Pay_
    button should become disabled.
 
-__START THIS TUTORIAL BY CLONING THE FOLLOWING JS BIN__:
+__START THIS TUTORIAL BY CLONING THE FOLLOWING CODEPEN__:
 
-<a class="jsbin-embed" href="https://jsbin.com/rosuzit/1/embed?output">JS Bin on jsbin.com</a>
+<p class="codepen" data-height="265" data-theme-id="0" data-default-tab="css,result" data-user="cherifGsoul" data-slug-hash="omrJZy" data-pen-title="Credit Card Guide (Simple) [Starter]">
+  <span>See the Pen <a href="https://codepen.io/cherifGsoul/pen/omrJZy/">
+  Credit Card Guide (Simple) [Starter]</a> by Mohamed Cherif Bouchelaghem (<a href="https://codepen.io/cherifGsoul">@cherifGsoul</a>)
```

Comment:
This starter CodePen has the form in the HTML tab. I think we should probably remove it and start it fresh with just the Stripe script.

Reply:
Done!